adapter/redisquota/redisquota.go

```diff
@@ -0,0 +1,315 @@
+// Copyright 2017 Google Inc.
```

Comment:
I don't see in here how expiring vs. non-expiring quotas are handled. How does that work?